etest/docker/container.py

```diff
@@ -17,9 +17,27 @@ def commit(*args, **kwargs):
     return common.CLIENT.commit(*args, **kwargs)
 
 
-def create(*args, **kwargs):
+def create(overlay, *args, **kwargs):
```

Comment:
We should add type hints as we update these interfaces as well.  I'd love to get away from these strange wrapper interfaces for safety but just adding the type hint for the argument you're adding is sufficient for this pull request.

If it's not too bad, making this a pathlib.Path would be preferable but that has far reaching consequences so is probably better for a different pull request.

Reply:
``overlay`` is filled with ``overlay.directory`` on ``Test.run()``. This variable is a string. As such, I think it would be best to hint the type of ``overlay`` as ``: str``